src/src/Utils.js

```diff
@@ -7,4 +7,34 @@ export function delay(duration) {
   return new Promise((resolve) => {
     setTimeout(() => resolve(), duration);
   });
+} 
+
+export function composeUrl(...args) {
```

Comment:
Duplicate composeUrl function in src/src/SelfServiceApiClient.js as far as I can see